pubsub/azure/eventhubs/eventhubs.go

```diff
@@ -0,0 +1,157 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package eventhubs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	eventhub "github.com/Azure/azure-event-hubs-go"
+	"github.com/Azure/azure-event-hubs-go/eph"
+	"github.com/Azure/azure-event-hubs-go/storage"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/dapr/components-contrib/pubsub"
+	"github.com/dapr/dapr/pkg/logger"
+)
+
+const (
+	// metadata
+
```

Comment:
nit: unnecessary return

Reply:
I did not quite understand what "unnecessary return" means in this context (did you mean "space"?). I'd be happy to make changes if you can help clarify. Thanks!